meinberlin/assets/scss/components/_poll.scss

```diff
@@ -187,14 +192,13 @@ $checkbox-size: 20px;
 }
 
 .editpoll__question-container {
-    // display: block;
+    display: flex;
```

Comment:
from screensize 992px to 991px it looks off

Reply:
@khamui having trouble replicating also that's 1px difference, is that the size you mean?

Reply:
@phillimorland  i doublechecked and this occurs for me (in chrome), but not sure if that change was intended?

<img width="952" alt="Bildschirmfoto 2022-10-24 um 13 37 04" src="https://user-images.githubusercontent.com/52801593/197527661-6a313847-0db0-44f6-8a49-8abcb0885248.png">

before the questions were spanning all the way to the right edge?

<img width="934" alt="Bildschirmfoto 2022-10-24 um 13 39 07" src="https://user-images.githubusercontent.com/52801593/197527714-f06260be-8317-4f93-9b42-6620aa43d134.png">

Reply:
cheers for screen shots, change wasn't intended, @goapunk is doing some jest config updates right now, will look at this once that's pushed

Reply:
@khamui ready for another look, can't tag you again